src/pages/project/index.tsx

```diff
@@ -41,7 +41,7 @@ function renderTags(tags: string[]): React.ReactNode[] {
 
 function ProjectItem({ item }: { item: Project }) {
   return (
-    <Link key={item._id} to={`/projects/${item._id}/`}>
+    <Link key={item._id} to={`/projects/${item._id}?projectName=${item.name}`}>
```

Comment:
If you want to show project name in breadcrumb, is it the correct way to do that?
Maybe get project detail from api?